system/Commands/Utilities/Optimize.php

```diff
@@ -17,8 +17,8 @@
 use CodeIgniter\Autoloader\FileLocatorCached;
 use CodeIgniter\CLI\BaseCommand;
 use CodeIgniter\CLI\CLI;
-use CodeIgniter\Exceptions\RuntimeException;
 use CodeIgniter\Publisher\Publisher;
+use RuntimeException;
```

Comment:
We don't throw RuntimeException after v4.6. Use `CodeIgniter\Exceptions\RuntimeException` instead.